.projenrc.ts

```diff
@@ -0,0 +1,391 @@
+import * as fs from "fs";
```

Comment:
Delete this file?

Reply:
hmm, must have come from the merge? deleted